rocky/katalogus/templates/plugin_container_image.html

```diff
@@ -0,0 +1,123 @@
+{% load static %}
+{% load i18n %}
+
+<div>
+    <h2>{% translate "Container image" %}</h2>
+    <p>
+        {% translate "The container image for this Boefje is:" %} <a href="{{ plugin.oci_image }}">{{ plugin.oci_image }}</a>
+    </p>
+    <div class="fifty-fifty">
+        <div>
+            <h3>{% translate "Variants" %}</h3>
+            <p>
+                {% blocktranslate %}
+                    Boefje variants that use the same container image. For more
+                    information you can read the documentation.
+                {% endblocktranslate %}
+            </p>
+        </div>
+        {% if perms.tools.can_add_boefje %}
+            <div class="horizontal-view toolbar">
+                <a class="button ghost"
+                   href="{% url "boefje_variant_setup" plugin_id=plugin.id organization_code=organization.code %}"><span aria-hidden="true" class="icon ti-plus"></span>{% translate "Add variant" %}</a>
+            </div>
+        {% endif %}
+    </div>
+    {% if variants %}
+        {% if new_variant %}
+            <p class="confirmation"
+               role="group"
+               aria-label="{% translate "new variant confirmation" %}">
```

Comment:
There's been some discussion on whether the `role="group"` is helpful or harmful for notifications. We may be removing this recommendation from the Manon docs. I would advise removing it, as it has the potential to _improve_ the accessibility of something that is already accessible, but also has the potential to make it _less_ accessible.

```suggestion
            <p class="confirmation">
```

If you do want to keep it, consider changing the label to just "confirmation". The role and label are meant as a text alternative for the visual style of the notification. That is: "visually distinct green block" == "notification".

The same applies to other notifications, like the "no variants explanation" (either remove role and label or change label to just "explanation").